cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/AbstractConductingEquipmentConversion.java

```diff
@@ -251,6 +255,9 @@ public void convertToDanglingLine(int boundarySide, double r, double x, double g
             dl.addAlias(tn, Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TOPOLOGICAL_NODE);
             dl.setProperty(Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + dl.getId() + "." + CgmesNames.TOPOLOGICAL_NODE, tn); // TODO: delete when aliases are correctly handled by mergedlines
         });
+        Optional.ofNullable(connectivityNodeId(boundarySide)).ifPresent(cn ->
+            dl.addAlias(cn, Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.CONNECTIVITY_NODE)
```

Comment:
Can you just add "boundary" in the name?

Reply:
added `_Boundary` suffix for the related alias types.